### PR TITLE
Fix potential memory leak in pacemaker

### PR DESCRIPTION
**Pull request by clstrfsck**
Fix potential memory leak in pacemaker

### CODE REVIEW

File: radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import com.radixdlt.consensus.View;
+import com.radixdlt.consensus.validators.ValidationState;
+import com.radixdlt.consensus.validators.ValidatorSet;
+import com.radixdlt.crypto.ECDSASignature;
+import com.radixdlt.crypto.ECPublicKey;
+import com.radixdlt.crypto.Hash;
+import com.radixdlt.utils.Longs;
+
+/**
+ * Manages pending votes for various vertices.
```
Comment (flotothemoon) [inline radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java]: This is pending votes for new views, not vertices, right?

File: radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+@NotThreadSafe
+@SecurityCritical
+public final class PendingNewViews {
+	private static final Logger log = LogManager.getLogger();
+
+	private final Map<View, ValidationState> newViewState = Maps.newHashMap();
+	private final Map<ECPublicKey, View> previousNewView = Maps.newHashMap();
+
+	/**
+	 * Inserts a vote for a given vertex, attempting to form a quorum certificate for that vertex.
```
Comment (flotothemoon) [inline radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java]: This too would be view, not vertex, right?

**Review by flotothemoon — APPROVED** [review]
Looks good, though documentation needs updating. It irks me a bit that `PendingNewViews` duplicates a lot of what `PendingVotes` does. Maybe we'll find an underlying pattern to simplify this.

File: radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java (hunk 1 above)
Comment (clstrfsck) [reply radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java]: Fixed.

File: radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java (hunk 2 above)
Comment (clstrfsck) [reply radixdlt/src/main/java/com/radixdlt/consensus/liveness/PendingNewViews.java]: Fixed.

**Review by talekhinezh — APPROVED** [review]

**Pull request merged**